pkg/tcpip/stack/transport_demuxer.go

```diff
@@ -259,7 +259,7 @@ var loopbackSubnet = func() tcpip.Subnet {
 // deliverPacket attempts to find one or more matching transport endpoints, and
 // then, if matches are found, delivers the packet to them. Returns true if it
 // found one or more endpoints, false otherwise.
-func (d *transportDemuxer) deliverPacket(r *Route, protocol tcpip.TransportProtocolNumber, netHeader buffer.View, vv buffer.VectorisedView, id TransportEndpointID) bool {
+func (d *transportDemuxer) deliverPacket(r *Route, protocol tcpip.TransportProtocolNumber, netHeader buffer.View, vv buffer.VectorisedView, id TransportEndpointID, first bool) bool {
```

Comment:
Why do we care if it's the first time? Also I believe this is not required anymore with the latest changes there is only demuxer for the stack and not a per NIC one.

Reply:
> Why do we care if it's the first time?

Please refer to the commit log of this change for the reason.

> Also I believe this is not required anymore with the latest changes there is only demuxer for the stack and not a per NIC one.

Previously, I don't want to introduce a big change in netstack in this patchset, so proposed a quick fix. If it's fixed in latest code, I'll do another round of rebase, and drop this commit if it's not necessary.